packages/ast-spec/src/special/TSEnumBody/spec.ts

```diff
@@ -0,0 +1,8 @@
+import type { AST_NODE_TYPES } from '../../ast-node-types';
+import type { BaseNode } from '../../base/BaseNode';
+import type { TSEnumMember } from '../../element/TSEnumMember/spec';
+
+export interface TSEnumBody extends BaseNode {
+  type: AST_NODE_TYPES.TSEnumBody;
+  members: TSEnumMember[];
```

Comment:
```suggestion
  members: TSEnumMember[];
  parent: TSEnumDeclaration;
```

This holds true, right?

Reply:
Yeah!!